### PR TITLE
Jusplainmike hangbot

### PR DESCRIPTION
**Pull request by jusplainmike**
Added service to send messages over Hangouts using hangoutsbot api plugin.

https://github.com/hangoutsbot/hangoutsbot/wiki/API-Plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ _mqttwarn_ supports a number of services (listed alphabetically below):
 * [gss](#gss)
 * [gss2](#gss2)
 * [hipchat](#hipchat)
+* [hangbot](#hangbot)
```
Comment (jpmens) [inline README.md]: Please keep the alphabetical order (hangbot before hipchat)

File: README.md [hunk 2]
```diff
@@ -874,6 +875,19 @@ The notify parameter (True or False) trigger a user notification (change the tab
 
 ![Hipchat](assets/hipchat.png)
 
+### `hangbot`
```
Comment (jpmens) [inline README.md]: This description should also follow alphabetical order in the README

File: services/hangbot.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+__author__    = 'Bram Hendrickx'
+__copyright__ = 'Copyright 2016 Bram Hendrickx'
```
Comment (jpmens) [inline services/hangbot.py]: Is 2016 correct?

File: services/hangbot.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        'content': item.message
+    }
+    try:
+        srv.logging.debug("Sending to hangoutsbot")
+        url = "https://" + item.addrs[0] + ":" + item.addrs[1]
+        headers = {'content-type': 'application/json'}
+        requests.post(url, data = json.dumps(payload), headers = headers, verify=False)
+        srv.logging.debug("Successfully sent ifttt event")
+    except Exception, e:
+        srv.logging.warning("Cannot send ifttt event: %s" % (str(e)))
```
Comment (jpmens) [inline services/hangbot.py]: A bit too much copy/paste :) `s/iftt/___/g`

**Comment by jusplainmike** [comment]
The script was modified from the ifttt.py script. It's almost completely
copy paste in that sense,

**Comment by jpmens** [comment]
Make it obvious at the top in the credits, and you'll be ok

**Comment by jpmens** [comment]
I see it's there. Thanks. Closing this, continuingi n your new PR

**Pull request closed without merging**